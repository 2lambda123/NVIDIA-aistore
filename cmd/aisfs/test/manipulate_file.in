echo "some content" > $DIR/abc.txt
ls $DIR
cat $DIR/abc.txt
cat $DIR/txt.txt // FAIL "no such file or directory"

mv $DIR/abc.txt $DIR/def.txt // FAIL "function not implemented"
cp $DIR/abc.txt $DIR/def.txt

ls $DIR | sort

rm -f $DIR/abc.txt

cat $DIR/abc.txt // FAIL "no such file or directory"
cat $DIR/def.txt

rm -f $DIR/def.txt
ls $DIR
