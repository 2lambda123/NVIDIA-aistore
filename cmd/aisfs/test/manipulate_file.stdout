abc.txt
some content
abc.txt
def.txt
some content
